### PR TITLE
Change ui of chat message

### PR DESCRIPTION
**Pull request by Ferossgp**
Adds some UI changes as a part of #9469 further improvements require bigger refactor on message components.

### CODE REVIEW

File: src/status_im/ui/screens/chat/styles/message/message.cljs [hunk 1]
```diff
@@ -93,6 +93,10 @@
     {:align-self    :flex-start
      :padding-left  (if platform/desktop? 24 8)}))
 
+(def message-author-touchable
+  {:margin-left        12
+   :padding-horizontal 2})
```
Comment (errorists) [inline src/status_im/ui/screens/chat/styles/message/message.cljs]: @Ferossgp hey, my mistake. it should be vertical here if needed at all (text line height is 18, touchable is 22 so 2px padding would center it inside the touchable)

File: src/status_im/ui/screens/chat/styles/message/message.cljs [hunk 2]
```diff
@@ -136,23 +140,33 @@
 
 (styles/defn emoji-message
   [{:keys [incoming-group]}]
-  {:font-size 40
-   :desktop   {:line-height 46}
-   :margin-top (if incoming-group 4 0)})
+  {:font-size    40
```
Comment (errorists) [inline src/status_im/ui/screens/chat/styles/message/message.cljs]: is that emoji only font-size? 40 is hardcore, can we lower it to 28

Comment (Ferossgp) [reply src/status_im/ui/screens/chat/styles/message/message.cljs]: Yes, it is only emoji size - I can do it.

**Review by flexsurfer — APPROVED** [review]

**Comment by errorists** [comment]
hey, can we do something about those guys? :)

Screenshot 2020-01-09 at 09 04 25

**Comment by errorists** [comment]
• Rounded corners ✅ 
• Styling on authors, looks off. Touchable container is 22px high, inside is vertically centred text label with the author name, line height 18px. 8px margin between the touchable and the message bubble preceding it.
• Timestamp, almost got it, it's 1px too high ;) it should appear like the [baseline](https://en.wikipedia.org/wiki/Baseline_(typography)#/media/File:Typography_Line_Terms.svg) of the text and timestamps are positioned at the same level.
• with Emoji messages it's hard to tell since it's placed on a white background, the goal is to have the font-size larger which is correct now but placed on regular message bubbles like other messages. but it's ok for the WIP PR ✅
• Reply icon, did you use the glyph that's in the font? I replaced the icon which was always giving us headaches with positioning with the '↪' arrow that's in Inter, our font, followed by a regular space. it appears like it's using a graphic asset as the margin between it and username is too large.

**Comment by errorists** [comment]
Reply icons are correct now! ✅ 
Timestamp Y position is correct ✅ 
Author values are still incorrect

Screenshot 2020-01-10 at 12 15 57


Areas of the #9469 issue which are not fixed yet:

• 3. Make sure the minimum space in a line between a timestamp is 12 (is 6 now, I kinda like it but it's probably wiser to have a larger margin)
• 6. Make sure that the date headers and status indicators match the spec
• 7. Update the appearance of Emoji only messages
• 8. Update the appearance of Sticker messages
_• 9. Make sure messages grow horizontally to fill the available space correctly_ (fixed in next commit)
• 10. Update the appearance of long messages

**Comment by errorists** [comment]
Message author values are correct ✅ 
Update the appearance of Emoji only messages ✅ 
Make sure messages grow horizontally to fill the available space correctly ✅ 
I came to like the current space between the timestamp so no further changes are necessary there ✅ 

left in the issue, can be divided into separate PRs.
• 6. Make sure that the date headers and status indicators match the spec
• 8. Update the appearance of Sticker messages
• 10. Update the appearance of long messages

File: src/status_im/ui/screens/chat/styles/message/message.cljs [hunk 3]
```diff
@@ -46,11 +39,12 @@
...
-         {:color (if (and outgoing (not emoji?))
-                   colors/white-transparent-70
-                   colors/gray)}
+         {:line-height 10
+          :color       (if outgoing
+                         colors/white-transparent-70
+                         colors/gray)}
          (when justify-timestamp? {:position              :absolute
-                                   :bottom                7
+                                   :bottom                (+ 6 3) ; 6 Bubble bottom, 3 message baseline
```
Comment (flexsurfer) [inline src/status_im/ui/screens/chat/styles/message/message.cljs]: why don't just use `9` ? and left (+ 6 3) in comment?

**Review by flexsurfer — COMMENTED** [review]
do we really need all these fonts? and BETA fonts ?

**Comment by Ferossgp** [comment]
@flexsurfer I haven't looked if they are used, I updated only the fonts which were already present. I would suggest to cleanup them in a sperate PR in case it may affect some platforms or code.

**Comment by errorists** [comment]
> do we really need all these fonts?

@flexsurfer that's a great question. to be honest, we don't. we use the following font weights: 
400 (Regular), 500 (Medium), 700 (Bold), with matching italics.

BETA, they're not really 'beta' like an app :) for those open source fonts, it means that some contributors haven't provided languages like Cyrillic, Farsi etc They're safe to use, but yea, we don't use them yet in the app.

**Review by flexsurfer — APPROVED** [review]

**Comment by Serhy** [comment]
Tested with iOS and Android (v8.1 and v9), ready for merge!
@Ferossgp please mind my commit fixing e2e test when squashing

**Comment by errorists** [comment]
when merging pls don't close the original issue as there are some items left unresolved there 🙏

**Pull request merged**